src/dom-utils.ts

```diff
@@ -107,15 +107,17 @@ export function updateTransitionEndListener(
   action: 'add' | 'remove',
   listener: (event: TransitionEvent) => void
 ): void {
-  const method = `${action}EventListener` as
-    | 'addEventListener'
-    | 'removeEventListener';
-
-  // some browsers apparently support `transition` (unprefixed) but only fire
-  // `webkitTransitionEnd`...
-  ['transitionend', 'webkitTransitionEnd'].forEach((event) => {
-    box[method](event, listener as EventListener);
-  });
+  if(listener) {
```

Comment:
nit: there should be a space before the `(`. Did Prettier not run on commit?

Reply:
Good point. Not sure why it happened. I initially used VS Code prettifier, but the style was too different, so I committed it the way it was.

Reply:
If you can update it this should be good to go